app/mishki-index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.min.css">
+  <title>Mishka</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="nav--wrapper">
+        <img class="nav__logo logo" src="/app/imajes/jpg/mishka-logo-mobile.svg copy.png" alt="Мишка">
+        <button class="main-nav__togle" type="button"><?xml version="1.0" encoding="iso-8859-1"?>
+          <svg class="svg__icon" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+             width="35px" height="30px" viewBox="0 0 612 612" style="enable-background:new 0 255 255 255;" xml:space="preserve">
+            <g>
+              <path d="M604.267,194.727c4.257,0,7.733-3.644,7.733-7.733v-40.169c0-4.256-3.283-7.733-7.733-7.733H7.733
+                c-4.256,0-7.733,3.644-7.733,7.733v40.169c0,4.256,3.283,7.733,7.733,7.733H604.267z"/>
+              <path d="M0,326.084c0,4.256,3.283,7.733,7.733,7.733h596.533c4.256,0,7.733-3.645,7.733-7.733v-40.169
+                c0-4.284-3.283-7.733-7.733-7.733H7.733c-4.256,0-7.733,3.645-7.733,7.733V326.084z"/>
+              <path d="M0,465.175c0,4.256,3.283,7.733,7.733,7.733h596.533c4.256,0,7.733-3.645,7.733-7.733v-40.169
+                c0-4.256-3.283-7.732-7.733-7.732H7.733c-4.256,0-7.733,3.644-7.733,7.732V465.175z"/>
+            </g>
+          </svg>
+          </button>
+        </div>
+        <ul class="nav__list site-list">
+          <li class="site-list__item"><a href="#">Каталог товаров</a></li>
```

Comment:
```suggestion
          <li class="site-list__item">
              <a href="#">
                  Каталог товаров
              </a>
          </li>
```
намного легче читать вна2ре говорю)))